### DISCUSSION TITLE
I would like to know more about the theme used in [Apollo](https://apollo.primeng.org/), specifically regarding its code structure and file organization.

### DISCUSSION THREAD

**Discussion by PraveenPremasinghe** [primefaces/community#2964] [PrimeNG]
I would like to know more about the theme used in [Apollo](https://apollo.primeng.org/), specifically regarding its code structure and file organization.

Does it use PrimeNG components?
Is any state management implemented?
How is routing handled?
Is it possible to access a sample code or demo of [Apollo](https://apollo.primeng.org/) before purchasing?
Thank you!

**Comment by qwadrox** [comment]
@PraveenPremasinghe hi! 

There is a free template: https://github.com/primefaces/sakai-ng
Probably the paid templates are similar.
